helix-core/src/main/java/org/apache/helix/manager/zk/ZKHelixManager.java

```diff
@@ -245,6 +253,11 @@ public ZKHelixManager(String clusterName, String instanceName, InstanceType inst
 
     _stateListener = stateListener;
 
+    if (helixParticipantProperty != null) {
+      _helixParticipantProperty = helixParticipantProperty;
```

Comment:
No need?

Reply:
The logic here is that if user provides helixParticipantProperty, we will use the provided one. Otherwise, we create a HelixParticipantProperty with default value.